pallets/automation-time/src/tests.rs

```diff
@@ -2364,6 +2446,7 @@ fn missed_tasks_removes_completed_tasks() {
 			vec![40],
```

Comment:
I created a #396 to track this test case not accurate issue in order to fix it later. It shouldn’t block this PR.